libs/sdk-ui-ext/src/insightView/InsightView.tsx

```diff
@@ -265,6 +265,7 @@ class InsightViewCore extends React.Component<IInsightViewProps & WrappedCompone
                         backend={this.props.backend}
                         colorPalette={this.props.colorPalette ?? this.state.colorPalette}
                         config={this.props.config}
+                        execConfig={this.props.execConfig}
```

Comment:
What about DashboardVIew, should we support it there too?

Reply:
IMO we don't need to support it in dashboard view as in the current sampling does not get saved, therefore it is not propagated in KD.

Reply:
let me confirm it with Ondra Macek

Reply:
But you might want to see a dashboard with a sampling enabled on all insight for example